wallet/src/de/schildbach/wallet/offline/AcceptBluetoothThread.java

```diff
@@ -76,7 +77,7 @@ public void run() {
                         is.readFully(msg);
 
                         try {
-                            final Transaction tx = new Transaction(Constants.NETWORK_PARAMETERS, msg);
+                            final Transaction tx = Transaction.read(ByteBuffer.wrap(msg));
```

Comment:
Check if this is the same as `new Transaction(Constants.NETWORK_PARAMETERS, msg)`